imap/jmap_calendar.c

```diff
@@ -5345,6 +5352,31 @@ static void remove_itip_messages(struct caldav_db *db,
     }
 }
 
+static int is_standalone(struct jmap_caleventid *eid,
```

Comment:
This function name is likely to misunderstand, as it returns a true-ish value on error. Would `check_eventid_exists` be more clear?